src/main/webapp/courseLoader.js

```diff
@@ -0,0 +1,124 @@
+google.charts.load("current", { packages: ["corechart"] });
+google.charts.setOnLoadCallback(makeGraphs);
+
+function fillTitles() {
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    const courseName = urlParams.get("course-name");
+    const term = urlParams.get("term");
+    document.getElementById("course-name").innerHTML = courseName;
+    document.getElementById("term-name").innerHTML = term;
+}
+
+function populateData(){
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    const courseName = urlParams.get("course-name");
+    const term = urlParams.get("term");
+    const profName = urlParams.get("prof-name");
+    const units = urlParams.get("num-units")
+    const schoolName = urlParams.get("school-name")
+
+  fetch(`/term-live?school-name=${schoolName}&course-name=${courseName}&term=${term}&prof-name=${profName}&num-units=${units}`)
+    .then((response) => response.json())
+    .then((data => {
+        makeGraph(data);
+    }));
+}
+
+function makeGraphs(dataObject){
+    const tempHoursList = dataObject.hoursList;
+    const hoursList = [["hours"],[3],[8]].concat(tempHoursList);
```

Comment:
What does "[3], [8]" do? Maybe add a comment if it's something special having to do with making the graphs. Same below.

Reply:
Because this is working of live data there database is currently empty due to the bug from the morning. I just put numbers there so something will show. I will comment them